src/cmd/services/m3coordinator/main/main.go

```diff
@@ -26,17 +26,17 @@ import (
 	"os"
 
 	"github.com/m3db/m3/src/query/server"
+	xconfig "github.com/m3db/m3/src/x/config"
 	"github.com/m3db/m3/src/x/etcd"
 )
 
-var (
-	configFile = flag.String("f", "", "configuration file")
-)
+var configFiles xconfig.StringSlice
 
 func main() {
+	flag.Var(&configFiles, "f", "configuration file(s)")
```

Comment:
What do you think about pulling all this configFiles stuff into `xconfig.StringSlice`, something like

`xconfig.ParseConfigFilenames(flag string, description string) ([]string, bool)`

That could handle the validation and make it a little easier to use? Don't mind either option much though, so up to you here :+1:

Reply:
I'd personally leave it like this for now.